src/handler.ts

```diff
@@ -24,9 +24,7 @@ if (getEnvValue("DD_TRACE_ENABLED", "true").toLowerCase() === "true") {
 
   // Configure the tracer to ignore HTTP calls made from the Lambda Library to the Extension
   tracer.use("http", {
-    // blacklist has been deprecated in favor of blocklist, but we still use it here for
-    // compatibility with older versions of the tracer
-    blacklist: /:8124/,
+    blocklist: /8124\/lambda/,
```

Comment:
do we want to leave the `:` to ensure that a customer who has an url with uuid/lambda (where uuid would finish with 8124) won't have any conflict? It seems unlikely but having the `:` would be more safe? WDYT?

Reply:
Sure, why not?